floris/optimization/load_optimization/load_optimization.py

```diff
@@ -0,0 +1,459 @@
+    farm_power = fmodel.get_farm_power()
+    values = fmodel.wind_data.values
+    return farm_power * values
+
+
+def compute_farm_voc(
+    fmodel: FlorisModel,
+    A: float,
+    load_ambient_tis: np.array,
+    wake_slope: float = 0.3,
+    max_dist_D: float = 10.0,
+    exp_ws_std: float = 1.0,
+    exp_thrust: float = 1.0,
+):
+    """Compute the farm Variable Operating Cost (VOC) for each findex.
+
+    Args:
+        fmodel (FlorisModel): FlorisModel object
+        A (float): Coefficient for the VOC calculation
+        load_ambient_tis (list or np.array): Ambient 'load' turbulence intensity for each findex
+        wake_slope (float, optional): Wake slope. Defaults to 0.3.
+        max_dist_D (flat, optional): Maximum distance in rotor diameters. Defaults to 10.0.
+        exp_ws_std (float, optional): Exponent for the wind speed standard deviation.
+            Defaults to 1.0.
+        exp_thrust (float, optional): Exponent for the thrust. Defaults to 1.0.
+
+    Returns:
+        np.array: Array of farm VOC for each findex
+
+    """
+    turbine_voc = compute_turbine_voc(
+        fmodel=fmodel,
+        A=A,
+        load_ambient_tis=load_ambient_tis,
+        wake_slope=wake_slope,
+        max_dist_D=max_dist_D,
+        exp_ws_std=exp_ws_std,
+        exp_thrust=exp_thrust,
+    )
+    return np.sum(turbine_voc, axis=1)
+
+
+def compute_net_revenue(
+    fmodel: FlorisModel,
+    A: float,
+    load_ambient_tis: np.array,
+    wake_slope: float = 0.3,
+    max_dist_D: float = 10.0,
+    exp_ws_std: float = 1.0,
+    exp_thrust: float = 1.0,
+):
+    """Compute the net revenue for the current layout."
+
+    Args:
+        fmodel (FlorisModel): FlorisModel object
+        A (float): Coefficient for the VOC calculation
+        load_ambient_tis (list or np.array): Ambient 'load' turbulence intensity for each findex
+        wake_slope (float, optional): Wake slope. Defaults to 0.3.
+        max_dist_D (flat, optional): Maximum distance in rotor diameters. Defaults to 10.0.
+        exp_ws_std (float, optional): Exponent for the wind speed standard deviation.
+            Defaults to 1.0.
+        exp_thrust (float, optional): Exponent for the thrust. Defaults to 1.0.
+
+    Returns:
+        np.array: Array of net revenue for each findex
+
+    """
+
+    revenue = compute_farm_revenue(
+        fmodel=fmodel,
+    )
+
+    farm_voc = compute_farm_voc(
+        fmodel=fmodel,
+        A=A,
+        load_ambient_tis=load_ambient_tis,
+        wake_slope=wake_slope,
+        max_dist_D=max_dist_D,
+        exp_ws_std=exp_ws_std,
+        exp_thrust=exp_thrust,
+    )
+
+    return revenue - farm_voc
+
+
+def find_A_to_satisfy_rev_voc_ratio(
+    fmodel: FlorisModel,
+    target_rev_voc_ratio: float,
+    load_ambient_tis: np.array,
+    wake_slope: float = 0.3,
+    max_dist_D: float = 10.0,
+    exp_ws_std: float = 1.0,
+    exp_thrust: float = 1.0,
+):
+    """Find the value of A that satisfies the target revenue to VOC ratio.
```

Comment:
I think this function needs a bit more documentation to explain what the `target_rev_voc_ratio` is and why one might want to find a value of A to meet that target. I think you mentioned this was useful for initializing the optimizer?